packages/renderer/src/stores/navigation/navigation-registry.ts

```diff
@@ -69,6 +70,7 @@ const init = (): void => {
   values.push(createNavigationKubernetesGroup());
   values.push(createNavigationExtensionEntry());
   values.push(createNavigationExtensionGroup());
+  values.push(createNavigationNetworkEntry());
```

Comment:
suggestion (blocking): we should move up this line, to avoid having the network entry bellow the extensions webviews, I would say before Kubernetes is probably the right place.

<img width="130" height="602" alt="Image" src="https://github.com/user-attachments/assets/9c19c8ba-8fca-4b9d-b479-3e0926d27efc" />